data_updates/Python/oecd_data.py

```diff
@@ -0,0 +1,127 @@
+import os
+import glob
+import numpy
+from numpy import NaN
+import pandas as pd
+from github import Github, InputGitTreeElement
+from django.conf import settings
+
+PURPOSE_CODE_TRENDS_URL = "https://staging-ddw.devinit.org/api/export/1241/"
+CSV_FOLDER = "oecd_csv"
+DATA_REPO = "devinit/di-website-data"
+REMOTE_BRANCH = "main"
+REMOTE_FOLDER = "2022"
+ODA_AID_TYPE_URL = "https://staging-ddw.devinit.org/api/export/1238/"
+ODA_CHANNEL_TYPE_URL = "https://staging-ddw.devinit.org/api/export/1237/"
+ODA_RECIP_TYPE_URL = "https://ddw.devinit.org/api/export/1339/"
```

Comment:
For the constants here that may need changing... Can we find a better way to handle them? You have hard-coded the URLs for example and they are staging URLs, are they the same ones that will be used on production? The Remote folder shows 2022, shall we maintain 2022 next year as well or we shall update it to 2023? If the year will keep changing, I propose using the datetime functions to get the current year. For the URLs, possibly we use Environment variables and fall back to the staging URLs if no environment variable has been found.